### DISCUSSION TITLE
Crashlytics: JavaScript stack traces not show on the Firebase console

### DISCUSSION THREAD

**Discussion by oferRounds** [invertase/react-native-firebase#5478] [General]
Hi, we’re using react-native-firebase version 12.1.0 + react native 0.63.4. On our Firebase console, the JS stack trace is not shown
What could be the reason?

Would be happy for your help!
Thanks!

**Comment by mikehardy** [comment]
Without more information I'm not exactly sure, but I am about 99% sure that whatever the reason is, it will be explained here: https://invertase.io/blog/react-native-firebase-crashlytics-configuration

Hope this helps?

**Comment by oferRounds** [comment]
Thanks @mikehardy! I have read this. The thing I saw that might relate to this is the option of `crashlytics_is_error_generation_on_js_crash_enabled`. On my case, I didn’t set it explicitly, as it seem it’s default to `true`. Am I right?
Anyhow, I can try to set it explicitly to `true`

**Reply by mikehardy** [reply]
It should default to true yes, I'll be curious if setting it explicitly changes things for you, that would hint at some sort of default-setting error

**Comment by oferRounds** [comment]
Ok, I’ve changed and I’ll see if it makes any different
Perhaps Mike - you have an example image of would I see the JS trace in the firebase console? Maybe I’m missing it

**Reply by mikehardy** [reply]
Like so: 
![image](https://user-images.githubusercontent.com/782704/124490535-d3d91c80-dd77-11eb-9e9a-b8c1278c20cf.png)

If you pull the e2e test app here (https://github.com/invertase/react-native-firebase/blob/master/tests/README.md) and run it, the app itself has a pressable button for a native crash or a javascript crash, and you can play with the firebase.json settings to trigger each. Same can (and should really?) be added to your work project behind a debug-mode check though, so you can really test your integration

**Reply by mikehardy** [reply]
(note, that crash is without hermes integrated, hermes stacks look different, but are still clearly javascript errors)

**Comment by oferRounds** [comment]
Thanks again Mike!
If I see something like, it means it’s not working for me?
![image](https://user-images.githubusercontent.com/1477963/124497174-27248c80-ddc3-11eb-9ee6-92010612a0b6.png)

**Reply by mikehardy** [reply]
"JavaScriptError" is the clear marking that this is in your javascript code, and the stack frames are into a jsbundle - this is definitely a javascript stack trace. It's working.

Your next job is to decode the thing - e.g., discussion here (with practical examples near the end) https://github.com/invertase/react-native-firebase/issues/3760

**Reply by oferRounds** [reply]
Nice! Good to know!
Ok, will look into it, thank you!!
